test/mpi/dtpools/configure.ac

```diff
@@ -0,0 +1,43 @@
+#                                               -*- Autoconf -*-
+# Process this file with autoconf to produce a configure script.
```

Comment:
All files should have top-level COPYRIGHT information.

Reply:
Fixed